core/modules/views/plugins/views_plugin_style_grid.inc

```diff
@@ -12,17 +12,20 @@
  */
 class views_plugin_style_grid extends views_plugin_style {
   /**
-   * Set default options
+   * Set default options.
    */
   function option_definition() {
     $options = parent::option_definition();
-
     $options['columns'] = array('default' => '4');
     $options['alignment'] = array('default' => 'horizontal');
-    $options['fill_single_line'] = array('default' => TRUE, 'bool' => TRUE);
-    $options['summary'] = array('default' => '');
-    $options['caption'] = array('default' => '');
-
+    $options['wrapper_classes'] = array('default' => '');
+    $options['column_classes'] = array('default' => 'views-column');
+    $options['first_column_classes'] = array('default' => 'views-column-first');
+    $options['last_column_classes'] = array('default' => 'views-column-last');
+    $options['row_classes'] = array('default' => 'views-row');
+    $options['first_row_classes'] = array('default' => 'views-row-first');
+    $options['last_row_classes'] = array('default' => 'views-row-last');
+    $options['default_classes'] = array('default' => 0);
```

Comment:
`$options['default_classes']` is a Boolean. See `if (!$options['default_classes'])` in `template_preprocess_views_view_grid()`